packages/x-license/src/verifyLicense/verifyLicense.ts

```diff
@@ -178,7 +178,7 @@ export function verifyLicense({
     return { status: LICENSE_STATUS.Invalid };
   }
 
-  if (license.planVersion === 'initial') {
+  if (license.planVersion === 'initial' && license.scope === 'pro') {
     // 'charts-pro' or 'tree-view-pro' can only be used with a newer license
     if (productScope === 'charts' || productScope === 'tree-view') {
```

Comment:
Suggestion to be more robust to upcomming changes. Otherwise each new product will have to be added in this condition

```suggestion
  const initialProductScopes = ['data-grid', 'date-pickers'];
  if (license.planVersion === 'initial' && license.scope === 'pro') {
    // 'charts-pro' or 'tree-view-pro' can only be used with a newer license
    if (!initialProScopes.includes(productScope)) {
```

Reply:
I renamed `productScope` into `productLine` because I think it can quickly become confusing to have `licenseScope` and `productScope` given that we often shortcut the `licenseScope` into `scope`

Reply:
And I applied your proposal :+1:

Reply:
I did some changes to rely on the package name instead of creating the `licenseScope` and `productScope` abstractions that don't bring a lot of value and add a lot of complexity to the code.

I'd be curious to have your opinion.